assets/src/edit-story/components/richText/getFontVariants.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { ITALIC, WEIGHT } from './customConstants';
+import { styleToNumeric } from './formatters/util';
+import { getAllStyleSetsInSelection } from './draftUtils';
+import { getSelectAllStateFromHTML } from './htmlManipulation';
+
+/**
+ * Gets font styles for a characters, considers italic and weight only.
+ *
+ * @param {Object} styles Set of styles.
+ * @return {[]} Array of found styles for the character.
+ */
+function getFontStylesForCharacter(styles) {
+  return styles
+    .toArray()
+    .map((style) => style.style ?? style)
+    .filter((style) => style.startsWith(ITALIC) || style.startsWith(WEIGHT));
```

Comment:
Similarly here, as style will be identical to `ITALIC` if anything.

```suggestion
    .filter((style) => style === ITALIC || style.startsWith(WEIGHT));
```